<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebDialog</class>
 <widget class="QDialog" name="WebDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basket</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Basket {
background-color: rgb(164, 164, 164);
border:1px solid #AAA;
border-radius:4px;

}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="Basket" native="true">
     <property name="styleSheet">
      <string notr="true">QLabel { color:rgb(20,20,20); }</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QTabWidget" name="tabWidget">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="tabPosition">
            <enum>QTabWidget::North</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="documentMode">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="packageFiles">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <attribute name="title">
             <string>Package Files</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="margin">
              <number>8</number>
             </property>
             <item row="0" column="0" colspan="2">
              <widget class="QListWidget" name="filesList">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="packageDetails">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget#webWidget {
	border:0px solid #555;
	border-radius:6px;

	background-color:rgba(122,122,122);
    color:rgba(20,20,20);

}</string>
            </property>
            <attribute name="title">
             <string>Package Details</string>
            </attribute>
            <layout class="QGridLayout" name="webLayout">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="noconnection">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>AppInfo Server is not reachable
Please check your network connection</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QWidget" name="webWidget" native="true">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="margin">
                 <number>4</number>
                </property>
                <item row="0" column="0">
                 <widget class="QWebView" name="webView">
                  <property name="styleSheet">
                   <string notr="true">background-color:rgba(0,0,0,0);</string>
                  </property>
                  <property name="url">
                   <url>
                    <string>about:blank</string>
                   </url>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget#widget{
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);

border-bottom:1px solid #CCC;
}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="packageName">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Package Details</string>
              </property>
              <property name="indent">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="cancelButton">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>26</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(20,20,20);
</string>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
